Description: add iopl() back so vmmouse_detect will work in kvm again.
Origin: upstream, reverses http://cgit.freedesktop.org/xorg/driver/xf86-input-vmmouse/commit/?id=bcdec3d0cd4434770cd841c33c030e0d7203881f
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=525039

diff -Nur xserver-xorg-input-vmmouse-12.6.5/tools/vmmouse_detect.c xserver-xorg-input-vmmouse-12.6.5.new/tools/vmmouse_detect.c
--- xserver-xorg-input-vmmouse-12.6.5/tools/vmmouse_detect.c	2009-05-11 12:59:48.000000000 -0400
+++ xserver-xorg-input-vmmouse-12.6.5.new/tools/vmmouse_detect.c	2010-03-05 09:53:21.000000000 -0500
@@ -53,6 +53,11 @@
    signal(SIGSEGV, segvCB);
 
 #if defined __i386__ || defined __x86_64__ 
+   /*
+    * To access i/o ports above 0x3ff, we need to be in iopl(3).
+    */
+
+   iopl(3);
    if (VMMouseClient_Enable()) {
       VMMouseClient_Disable();
       return 0;
